test/masq.test.js

```diff
@@ -65,67 +81,52 @@ beforeEach(async () => {
 afterEach(async () => {
   await masq.signout()
   mockMasqApp.destroy()
-  common.utils.resetDbList()
+  resetDbList()
 })
 
 describe('localStorage and sessionStorage', () => {
   test('check that localStorage exists', () => {
     window.localStorage.setItem('testKey', 'testValue')
     expect(window.localStorage.getItem('testKey')).toBe('testValue')
+    window.localStorage.clear()
   })
 
   test('check that sessionStorage exists', () => {
     window.sessionStorage.setItem('testKey', 'testValue')
     expect(window.sessionStorage.getItem('testKey')).toBe('testValue')
+    window.sessionStorage.clear()
   })
 })
 
-describe('Test mock functions', () => {
+describe('Mock functions', () => {
   test('dbExists should work as expected', async () => {
-    expect(await common.utils.dbExists('db1')).toBe(false)
-    try {
-      await common.utils.createPromisifiedHyperDB('db1')
-      expect(await common.utils.dbExists('db1')).toBe(true)
-    } catch (error) {
-      console.log(error)
-    }
-    common.utils.resetDbList()
-    expect(await common.utils.dbExists('db1')).toBe(false)
-    await common.utils.createPromisifiedHyperDB('db1')
-    expect(await common.utils.dbExists('db1')).toBe(true)
+    expect(await dbExists('db1')).toBe(false)
+    await createPromisifiedHyperDB('db1')
+    expect(await dbExists('db1')).toBe(true)
+    resetDbList()
+    expect(await dbExists('db1')).toBe(false)
+    await createPromisifiedHyperDB('db1')
+    expect(await dbExists('db1')).toBe(true)
   })
 })
 
-async function logInWithMasqAppMock (stayConnected) {
-  // stop replication if logInWithMasqAppMock has already been called
-  mockMasqApp.destroy()
-
-  const link = await masq.getLoginLink()
-  const hashParams = common.utils.getHashParams(link)
-
-  await Promise.all([
-    mockMasqApp.handleConnectionAuthorized(hashParams.channel, hashParams.key),
-    masq.logIntoMasq(stayConnected)
-  ])
-}
-
-describe('Test login procedure', () => {
+describe('Login procedure', () => {
   test('should generate a pairing link', async () => {
     const uuidSize = 36
     const link = await masq.getLoginLink()
     const url = new URL(link)
     const base = config.MASQ_APP_BASE_URL
     expect(url.origin + url.pathname).toBe(base)
-    const hashParams = common.utils.getHashParams(link)
+    const hashParams = getHashParams(link)
     expect(hashParams.channel).toHaveLength(uuidSize)
   })
 
   test('should join a channel', async () => {
     expect.assertions(1)
```

Comment:
maybe use the same pattern as with the catch error, and remove expect.assertions and test a temporary value which should be assigned sw.peers by the end of the test ?